valhalla/jawn/src/lib/stores/experimentStore.ts

```diff
@@ -474,9 +472,11 @@ function getExperimentDatasetScores(
     const averageCost = validRows.length > 0 ? totalCost / validRows.length : 0;
 
     return ok({
-      dateCreated: new Date(latest.createdAt),
-      model: latest.model,
-      cost: averageCost,
+      scores: {
```

Comment:
nit: is `scores` ever going to be an array? If not, this should be renamed to `score`

Reply:
@ScottMktn it's Record with key <> value scores.

Reply:
I think we an keep this naming, we have the same for request headers which is Record<string,string>